tutorials/utils.py

```diff
@@ -0,0 +1,58 @@
+"""Utility funcitons for tutorials"""
+
+import numpy as np
+from scipy import signal
+
+def generate_trace(reflection_event_times=(20, 1400, 2400), reflection_event_amplitude=(6.0, -12.0, 8.0),
+                   nmo_velocity=(1.6, 2.0, 2.4), wavelet_lenght=50, wavelet_width=5, **kwargs):
+    """Generates a seismic trace using reflectivity parameters and trace's headers with Ricker impulse.
```

Comment:
```suggestion
    """Generates a seismic trace with Ricker impulse using reflectivity parameters and trace's headers.
```
Less confusing as for me